### PR TITLE
Prettier + ESLint Commit Hook

### PR DESCRIPTION
**Pull request by mrigdon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,5 +47,24 @@
...
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,ts,html,json,md,css}": [
+      "prettier --write",
+      "git add"
+    ],
+    "*.ts": [
+      "yarn type-check",
```
Comment (asha-cs) [inline package.json]: So this will just output any errors during the commit, but won't block the commit?

Comment (mrigdon) [reply package.json]: it will block the commit

Comment (asha-cs) [reply package.json]: Oh great!

Comment (mrigdon) [reply package.json]: but you can bypass the check by doing `git commit -m "foo" --no-verify`

**Pull request merged**